src/System.Text.Json/pkg/System.Text.Json.pkgproj

```diff
@@ -4,7 +4,9 @@
     <ProjectReference Include="..\ref\System.Text.Json.csproj">
       <SupportedFramework>net461;netcoreapp2.0;uap10.0.16299;$(AllXamarinFrameworks)</SupportedFramework>
     </ProjectReference>
-    <ProjectReference Include="..\src\System.Text.Json.csproj" />
+    <ProjectReference Include="..\src\System.Text.Json.csproj">
+      <SupportedFramework>net461;netcoreapp2.0;uap10.0.16299;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
Can you explain why we need to add the src project?

Reply:
The source project was already there, all we did here was adding the Supported frameworks clause to the source project. I did it because this project will not have the ref asset in the package because of the following property:

https://github.com/dotnet/corefx/blob/2469a91c2aa68f9cb6b0f18f75dffc673a1f7895/src/System.Text.Json/pkg/System.Text.Json.pkgproj#L19-L23

Since we have that, we do still want to make sure that we do validation even when only the src project is the one includded in the package, hence I added the supported clause to the source project.